### PR TITLE
fixes welding effects

### PR DESCRIPTION
**Pull request by tattax**
what the fuck is this shit why does it look so bad
i'm 100% the original port had this image and it was just so fucked up

BEFORE:
CWVaq50ep4

AFTER:
JGkZqSaMpk

one catgirl can really change everything

:cl:  
bugfix: above lightning layer now actually ignores lightning
imageadd: welding effects are fixed into not looking like shit
/:cl:

### CODE REVIEW

**Comment by tattax** [comment]
HMMM HRRRR GET OUT OF MY BRANCH @Moltijoe

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -102,7 +102,7 @@
 	taste_description = "slime"
 
 /datum/reagent/vaccine/reaction_mob(mob/living/L, method=TOUCH, reac_volume)
-	if(islist(data) && (method == INGEST || method == INJECT))
+	if(islist(data) && method == INJECT)
```
Comment (tattax) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]:
```suggestion
	if(islist(data) && (method == INGEST || method == INJECT))
```

**Comment by Moltijoe** [comment]
AHAHAHAHA, how the fuck did that even happen

**Comment by cuackles** [comment]
Oh yeah they should have been looking like that since long ago, it was just a error that wasn't fixed

**Comment by githubuser4141** [comment]
Aw this wasn't intentional? I always thought the top meshed with our artstyle better as we don't TG's more modern look

**Review by ToasterBiome — APPROVED** [review]

**Pull request merged**